File: c++/allSTL/Stack.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main(){
    
    stack<int> st;
    st.push(1765);
    st.push(1343);
    st.push(341);
    st.push(13);
    st.push(14);

    while(st.empty() == false){
        cout<<st.top()<<endl;
        st.pop();
    }

    return 0;
}

File: python/Stack.py
# Stack implementation using list

stack = []

# Push elements
stack.append(10)
stack.append(20)
stack.append(30)
print("Stack after pushes:", stack)

# Pop element
popped = stack.pop()
print("Popped element:", popped)
print("Stack after pop:", stack)

# Peek top element
print("Top element:", stack[-1])
